backend/routes/authRoutes.js

```diff
@@ -0,0 +1,102 @@
+require('dotenv').config();
```

Comment:
Make sure you have the eslint extension enabled in VSCode. There are a lot of eslint errors in your code.